app/controllers/api/v1/graphql_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class GraphqlController < ApplicationController
+      def create
+        render json: result
+      rescue StandardError => e
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/59338f029e412186a0a6dc405561dc7f2a762a99/app/controllers/api/v1/graphql_controller.rb#L8"></span>Has the variable name ‘e’</td>
    </tr>
  </tbody>
</table>